docs/source/platform/federation.mdx

```diff
@@ -0,0 +1,228 @@
+[//]: # (Assignee: Caydie)
+[//]: # (Reviewer: Jake)
+
+Federation allows teams to work independently on implementing services without needing to coordinate over a gigantic and all-encompassing schema. However, this increase in autonomy requires control to ensure that teams that operate on different services are respecting [defined dependencies](https://www.apollographql.com/docs/apollo-server/federation/federation-spec/). The Apollo GraphQL platform provides tools to help ensure that this increase in autonomy doesn't come at a cost to stability.
+
+In particular, along with [validating overall schema changes against known operations](https://www.apollographql.com/docs/platform/schema-validation/), running `apollo service:check` for an implementing service will ensure that the overall Graph still composes to a valid schema, and will output any violated dependencies if present.
+
+With a federated graph, use the `apollo service:check` command to validate individual service changes by adding the `--serviceName` flag. Engine will run composition on the proposed partial schema with the current list of implementing services and their partial schemas, making sure that the composition is successful. That composed schema will then be diff'ed against the most recently registered schema and validate that those changes are safe.
+
+
+> Using the `apollo service:push` and `apollo service:check` command to track changes to your graph
+
+<!--
+[//]: # (Description: An overview of how and when to use `apollo service:push` and `apollo service:check` in your CI/CD workflow, linking out to the specifics of usage in the specific docs (#502 and #503). Ideally this section includes some screenshots of the workflows in practice.)
+[//]: # (Assignee: James)
+[//]: # (Reviewer: Adam Z)
+-->
+
+## Running with managed federation
+> Taking control of how service changes bubble up to the Apollo Gateway
+
+[//]: # (Description: An overview of the concept of managed federation, linking out to the details on gateway defaults w/ an API key and the specfics on setting up the lifecycle options. Ideally this section has a diagram of managed federation lifecycle.)
+[//]: # (Assignee: Adam Z)
+[//]: # (Reviewer: Trevor)
+
+In small scale cases it can be sufficient to define the list of services comprising a federated graph directly within the gateway, and manually update the list and restart the server as services are modified and created. However for larger scale projects it is helpful to be able to remotely manage the gateway, such that new services may be added, or existing services modified, all with zero downtime.
+
+For these use cases, a gateway can be configured to pull a remotely hosted service list from the [Schema Registry][] on `Apollo Dashboard`. This is `Managed Federation`, and it is a key component of building a robust federated graph that can be concurrently developed by teams of any size.
+
+# Running Federation in Production
```

Comment:
take this introduction, and re-purpose it for the `Managed Federation` section that comes after `validating your schema`

The below section of "Best Practices" should remain as is and stand alone.